From e8a38297731a7686b03ce914784ad33c3f8a6a9a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Fri, 31 Oct 2025 15:45:57 +0100
Subject: [PATCH 035/133] std.Target: bump amdhsa max version to 7.1.0
X-Developer-Signature: v=1; a=openpgp-sha256; l=725; i=xtex@astrafall.org;
 h=from:subject; bh=8e1qEumpycSVr0Snel7ytzjq9y74NI2TftT5HgJZoh0=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cv8Gz6//bBEim+Cq97N7Bft716MxvisDRTYVV+exn
 g38ZzCrYyILgxgXg6WYIkuRYYM3q046v+iyclmYOaxMIEOkRRoYGBgYWBj4chPzSo10jPRMtQ31
 DA11jHSMGLg4BWCq5dkY/gd+/vzwbvxDy3q3Xr3z2okv9k3bOXmHb/OuFwvM0lWEjrkx/DOqXm5
 xapd39d91f7lL50itPuz7QJZNV/CCWXp424HvgvwA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit 802ee515d8819788efc82a4c5f5c6b14abd8fbf1)
---
 lib/std/Target.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/std/Target.zig b/lib/std/Target.zig
index 847a0689e68c..ce0c06efba67 100644
--- a/lib/std/Target.zig
+++ b/lib/std/Target.zig
@@ -615,7 +615,7 @@ pub const Os = struct {
                 .amdhsa => .{
                     .semver = .{
                         .min = .{ .major = 5, .minor = 0, .patch = 0 },
-                        .max = .{ .major = 6, .minor = 4, .patch = 2 },
+                        .max = .{ .major = 7, .minor = 1, .patch = 0 },
                     },
                 },
                 .amdpal => .{
-- 
2.52.0

